action.yaml

```diff
@@ -47,6 +47,10 @@ inputs:
     description: 'Whether to register the deployment (vs automatic deploy).'
     required: false
     default: 'false'
+  codedeploy_file_exists_behavior:
+    description: 'What to do if the file already exists in the deployment location.'
```

Comment:
Should we add `DISALLOW|OVERWRITE|RETAIN` into the description? Help with the other options.

Reply:
Good idea.